### DISCUSSION TITLE
Change Storage limit, self-hosted?

### DISCUSSION THREAD

**Discussion by ghost** [ente-io/ente#2193] [Q&A] [answered]
HI, 
I self-hosted Ente, but the app shows 5 GB as a storage limit. Also, it shows that I am in a test phase.
Is there a way to change this storage limit, as I am already hosting it myself?

**Comment by mnvr** [comment, marked as answer]
https://help.ente.io/self-hosting/faq/#how-do-i-increase-storage-space-for-users-on-my-self-hosted-instance

**Reply by ghost** [reply]
Ah ok sorry, I should have checked the FAQ first :D 

Thanks for the quick reply

**Comment by fbarbe00** [comment]
The link is dead now, for anyone else wondering, this seems to have now been moved here: https://help.ente.io/self-hosting/administration/cli#step-4-increase-storage-and-account-validity . The solution is to use the ente cli
